packages/aws-cdk-lib/aws-ecs/test/ec2/ec2-task-definition.test.ts

```diff
@@ -1110,6 +1110,67 @@ describe('ec2 task definition', () => {
         }],
       });
     });
+
+    test('correctly sets env variables when using EC2 capacity provider with AWSVPC mode', () => {
```

Comment:
Can you also add another test with `awsvpc` network mode and no added environment variables? (will set only `AWS_REGION`)